scripts/updateOrderPortal.py

```diff
@@ -27,7 +29,7 @@ def update_order_internal_id(self, open_date, dry_run, project_id):
             pjs = self.lims.get_projects(open_date=open_date.strftime("%Y-%m-%d"))
         for project in pjs:
             try:
-                pname = project.name
+                pass
```

Comment:
```suggestion
                pname = project.name
```

Reply:
This would amount to creating an unused variable, no?

Reply:
We are checking if the project is present in LIMS by trying to access it here. If we have `pass` here, the check is never made.

Reply:
Oh that's right! I guess that's why ruff calls it "unsafe" fixes 😆 never seen an example of it actually messing anything up, very good catch!

Reply:
I kept the statement raising the exception but removed the unused variable [24c3935](https://github.com/NationalGenomicsInfrastructure/LIMS2DB/pull/171/commits/24c393573b6de1220a7e6ecdf9e9aa4c777ac9d6)